l10n_it_statutory_financial_statements/readme/CONFIGURE.rst

```diff
@@ -0,0 +1 @@
+Se necessario (altrimenti vedi `l10n_it_bilancio_civilistico_mapping`) impostare su ogni conto il campo "raggruppo civilistico"
```

Comment:
Andrebbe usata la forma impersonale in tutta la frase.

"raggruppo civilistico" non lo trovo, si riferisce alla stringa "Civil code group"?
Se si, dovrebbe essere "Gruppo codice civilistico".

Se si intendeva invece "Codice gruppo civilistico",
s/Civil code group/Civil group code

Se si vuole mantenere la forma italiana "Gruppo civilistico" la stringa sorgente dovrebbe essere "Civil group"

Inoltre, come da linee guida, le descrizioni dovrebbero essere bilingue
https://www.odoo-italia.org/documentazione/sviluppo/guidelines.html#descrizione-bilingue

Reply:
Grazie, fatto